Library/Formula/gyp.rb

```diff
@@ -0,0 +1,24 @@
+class Gyp < Formula
+  desc "Generate Your Projects"
+  homepage "https://gyp.gsrc.io/"
+  url "https://chromium.googlesource.com/external/gyp",
+    :using => :git,
+    :revision => "7d29c3de1f5eb74330b81b78ea116b67f684b23b"
```

Comment:
You are right, there are still no tags. I just found some previously closed / rejected pull requests that have tried to do the same, so I guess this one might also not be merged. That's ok and my fault, as I have not searched for them before submitting the pull request.

Do you see any chance to get a formula for `gyp`? Maybe a fork of the official project that just adds tags and tarballs or similar? I am open for all ideas. Thanks!